src/main/resources/api/api.yaml

```diff
@@ -256,6 +309,33 @@ definitions:
   #
   # Quizzes
   #
+  V1Quiz:
+      required:
+        - id
+        - startedAt
```

Comment:
How do you think are startedAt finishedAt and  testee always available? For example after quiz generation.